<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="46" total="47" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-08-11T12:55:54 IST" name="Default suite" finished-at="2023-08-11T12:56:45 IST" duration-ms="50205">
    <groups>
    </groups>
    <test started-at="2023-08-11T12:55:54 IST" name="Default test" finished-at="2023-08-11T12:56:45 IST" duration-ms="50205">
      <class name="com.test.CreateMeetingTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:com.test.CreateMeetingTest@2925bf5b]" started-at="2023-08-11T12:55:54 IST" name="launchApplication" finished-at="2023-08-11T12:56:03 IST" duration-ms="8887" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="mandatoryFieldsOnly(java.util.HashMap)[pri:0, instance:com.test.CreateMeetingTest@2925bf5b]" started-at="2023-08-11T12:56:04 IST" name="mandatoryFieldsOnly" data-provider="getData0" finished-at="2023-08-11T12:56:45 IST" duration-ms="40483" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{date=11, meetingnameAssert=Legal Case 5 - Creating legal briefs, meetingName=11:00 AM Legal Case 5 - Creating legal briefs, scrollTime=14, comment=0.mandatoryFieldsOnly, taskName=Creating legal briefs , startTime= 11:00 , endTime= 11:30 , matterName=Legal Case 5 , monthAndTime=Aug 11 2023 : 11:00 AM - 11:30 AM}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [ Aug 11 2023 : 11:00 AM - 11:30 AM] but found [Aug 11 2023 : 11:00 AM - 11:30 AM]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [ Aug 11 2023 : 11:00 AM - 11:30 AM] but found [Aug 11 2023 : 11:00 AM - 11:30 AM]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.pom.ViewMeetingsPOM.assertMonthandTime(ViewMeetingsPOM.java:163)
at com.test.CreateMeetingTest.mandatoryFieldsOnly(CreateMeetingTest.java:38)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mandatoryFieldsOnly -->
        <test-method is-config="true" signature="down()[pri:0, instance:com.test.CreateMeetingTest@2925bf5b]" started-at="2023-08-11T12:56:45 IST" name="down" finished-at="2023-08-11T12:56:45 IST" duration-ms="12" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- down -->
      </class> <!-- com.test.CreateMeetingTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
